CHANGELOG.md

```diff
@@ -1,17 +1,16 @@
 ## 0.76.0 / 2025-08-08
 
-[FEATURE] OAuth2 TLSConfig and ProxyConfig settings. #6735
-[ENHANCEMENT] Use service endpoint role for alerting. #6784
-[FEATURE] Support configuring OAuth2 in ScrapeConfig. #6814
-[FEATURE] Add scale subresource to Alertmanger resource. #6728
-[ENHANCEMENT] set the namespace if empty for alertmanagers to the current namespace in Prometheus CR. #6338
-[ENHANCEMENT] support TLSConfig TLSVersion setting. #6736
-[ENHANCEMENT] optimize `TLSConfig` to `addSafeTLSToYaml`. #6726
-[ENHANCEMENT] Add ServiceDiscoveryRole configuration for use of Endpoints vs EndpointSlice. #6672
-[ENHANCEMENT] Add secret label selector to filter secrets to watch. #6731
-[CHANGE] Add listType:=set to Prometheus secrets field. This allows for having a different manager for each secret. #6762
-[ENHANCEMENT] Add attachMetadata field to ScrapeClasses. #6756
-[FEATURE] Add ScaleWay service discovery to the ScrapeConfig CRD. #6711
+[CHANGE] Change the `secrets` field of Prometheus and PrometheusAgent CRDs from `atomic` to `set`. This allows different managers to manage entries separately. #6762
```

Comment:
As a user who has no idea about developments, I feel this entry is still confusing. What is atomic? what is set? Why should they care? `This allows different managers to manage entries separately`, what entries 😅?

It's categorized as [CHANGED], so what behavior is being changed here?

I'll try to do some investigation, because I also don't understand completely what is changing here 😬

Reply:
I tend to agree with you, this is confuse and after reading this, I don't think it's a change will impact user and should be on the change log, I'll remove it.